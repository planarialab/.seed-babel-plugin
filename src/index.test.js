// @flow
import plugin from '.'
import pluginTester from 'babel-plugin-tester'

// jest.mock('./')

// const code = ``

// $FlowFixMe
// gist.mockImplementation(() => code)

// afterAll(() => {})

pluginTester({
  title: 'default',
  plugin,
  snapshot: true,
  tests: [
    `const oof = 1`,
    {
      title: 'test title',
      code: `const foo = 1`,
    },
  ],
})
