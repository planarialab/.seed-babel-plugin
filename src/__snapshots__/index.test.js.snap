// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`1. seed 1`] = `
"
const oof = 1

      ↓ ↓ ↓ ↓ ↓ ↓

const foo = 1;
"
`;

exports[`test title 1`] = `
"
const foo = 1

      ↓ ↓ ↓ ↓ ↓ ↓

const oof = 1;
"
`;
